apps/src/templates/sectionProgress/standards/ProgressBoxForLessonNumber.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import ProgressBox from '../ProgressBox';
+
+const styles = {
+  lessonBox: {
+    marginRight: 10
+  }
+};
+
+export default class ProgressBoxForLessonNumber extends Component {
+  static propTypes = {
+    completed: PropTypes.bool,
+    lessonNumber: PropTypes.number
+  };
+
+  render() {
+    const {completed, lessonNumber} = this.props;
+
+    return (
+      <ProgressBox
+        style={styles.lessonBox}
+        started={completed}
+        incomplete={completed ? 0 : 20}
+        imperfect={0}
+        perfect={completed ? 20 : 0}
+        showLessonNumber
```

Comment:
I like this wrapper! 

You won't need `showLessonNumber` anymore since I removed it in #32429 because we can use the presence of `lessonNumber` to decide whether to show the number or not.

Reply:
great! removed!